xc/xenops_server_xen.ml

```diff
@@ -728,7 +728,7 @@ module VM = struct
 		| Reboot -> Domain.Reboot
 		| PowerOff -> Domain.PowerOff
 		| Suspend -> Domain.Suspend
-		| Halt -> Domain.Halt
+		| Halt -> Domain.PowerOff
```

Comment:
This is going to be very confusing. We should leave this translation as it is, and solve the problem at the source: the implementation of the `VM_shutdown` operation in xenops_server.ml. This is where `Halt` is coming from:

    (Opt.default [] (Opt.map (fun x -> [ VM_shutdown_domain(id, Halt, x) ]) timeout)

We should just change that to `PowerOff`.